tests/Dotenv/DotenvTest.php

```diff
@@ -47,6 +47,14 @@ public function testCommentedDotenvLoadsEnvironmentVars()
         $this->assertEmpty(getenv('CNULL'));
     }
 
+    public function testDotenvLoadsEnvironmentVarsWithoutTrimmingQuoted()
+    {
+        $dotenv = new Dotenv($this->fixturesFolder, 'commented.env', false);
+        $dotenv->load();
+        $this->assertSame('bar', getenv('CFOO'));
```

Comment:
note how this variable should not have the leading spaces because it wasn't quoted, but the other one will have trailing spaces because it was quoted